src/content/docs/en/editor-setup.mdx

```diff
@@ -24,11 +25,14 @@ import ReadMore from '~/components/ReadMore.astro';
 
 <ReadMore>See how to [set up TypeScript](/en/guides/typescript/) in your Astro project.</ReadMore>
 
+## Zed
+
+[Zed](https://zed.dev/) is a newly open sourced code editor that added support for Astro in version 0.123.2. You can install the [Astro extension](https://github.com/zed-industries/zed/tree/main/extensions/astro) in the IDE's Extensions tab. This extension includes features like syntax highlighting, code completion, and formatting.
```

Comment:
```suggestion
[Zed](https://zed.dev/) is an open-source code editor that added support for Astro in version 0.123.2. You can install the [Astro extension](https://github.com/zed-industries/zed/tree/main/extensions/astro) in the IDE's Extensions tab. This extension includes features like syntax highlighting, code completion, and formatting.
```
Saying anything about "new" means it will get out of date at some point and is extra maintenance :smile:

If it's open-source NOW, we can just say that.